src/components/sections/AdvancedResources.tsx

```diff
@@ -0,0 +1,23 @@
+import Block from "../Block";
+const AdvancedResourcesSection = ({ themes, filteredData }: any) => {
```

Comment:
Can you change this to ResourcesSection as it will be used  for both resource pages (advance and basic)

Reply:
fixed